README.md

```diff
@@ -1,32 +1,127 @@
+<h1 align="center">
+  <br>
+  <a href="https://github.com/Mudlet/Mudlet"><img src="https://bazaar.launchpad.net/~mudlet-makers/mudlet/artwork/download/head:/mudlet_main_256px.pn-20171109060211-hidv2woj9dffwyql-11/mudlet_main_256px.png" alt="Mudlet"></a>
+  <br>
+  Mudlet
+  <br>
+</h1>
+
+<h4 align="center">Play immersive, multiplayer, pure-text RPGs on Mudlet.</h4>
+
+<p align="center">
+  <a href="http://mudlet.org/download/">
+    <img src="https://travis-ci.org/Mudlet/Mudlet.svg?branch=development"
+         alt="Build status">
+  </a>
+  <a href="https://discord.gg/kuYvMQ9">
+    <img src="https://img.shields.io/badge/discord-join%20chat-7289DA.svg">
+  </a>
+  <a href="https://github.com/mudlet/mudlet/issues">
+    <img src="https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat">
+  </a>
+  <a href="https://github.com/Mudlet/Mudlet/stargazers">
+    <img src="https://img.shields.io/github/stars/Mudlet/Mudlet.svg"/>
+  </a>
+</p>
+
+<p align="center">
+  <a href="#about">About</a> •
+  <a href="#key-features">Key Features</a> •
+  <a href="#how-to-use">How To Use</a> •
+  <a href="#download">Download</a> •
+  <a href="#credits">Credits</a> •
+  <a href="#roadmap">Roadmap</a> •
+  <a href="#license">License</a>
+</p>
+
+<p align="center">
+  <img alt="Interfaces players have created with Mudlet" height="500" src="https://bazaar.launchpad.net/~mudlet-makers/mudlet/artwork/download/head:/samplemudletuis.gif-20190701193442-s3gex5fzreha37jz-1/sample-mudlet-uis.gif">
+</p>
+
+## About
 
-# Mudlet
+Mudlet is a quality MUD client, designed to take mudding to a new level.
 
-[![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/31fqq844br6g6i40/branch/development?svg=true)](https://ci.appveyor.com/project/Mudlet/mudlet/branch/development) [![GitHub Build Status](https://travis-ci.org/Mudlet/Mudlet.svg?branch=development)](https://travis-ci.org/Mudlet/Mudlet) [![GitHub stars](https://img.shields.io/github/stars/Mudlet/Mudlet.svg)](https://github.com/Mudlet/Mudlet/stargazers) [![Discord](https://img.shields.io/badge/discord-join%20chat-7289DA.svg)](https://discord.gg/kuYvMQ9)
+It’s a modern breed of a client on the gaming scene – with an intuitive user interface, a specially designed scripting framework, and a very fast text display. That, along with cross-platform capability and an open-source development model result in a very likable game client.
 
-Mudlet is a quality MUD client, designed to take mudding to a new level.
+## Key Features
 
-It’s a new breed of a client on the MUD scene – with an intuitive user interface, a specially designed scripting framework, and a very fast text display. That, along with cross-platform capability and an open-source development model result in a very likable MUD client.
+* Simple to use
+* Very, very fast
+* Active development
+* Big, thriving community
+* Poweful trigger engine
+* Backed by a thorough Lua engine
```

Comment:
What is a "thorough" scripting engine?

Reply:
Trying to say that we have a ton of functionality in the Lua API

Reply:
While trying to make people who don't know what an 'API' stands for understand what it means

Reply:
I read it as "profound" or "exhaustive"

Reply:
I think I got the gist, but the usage was unclear to me. But maybe it's just my lack of understanding :grin: Go on then!

Reply:
Yeah I think every technically-minded person will find it confusing... but I can't say "powerful API" again, hmm

Reply:
Reworded